docs/source/command_line.rst

```diff
@@ -766,6 +766,9 @@ of the above sections.
     Note: the exact list of flags enabled by running :option:`--strict` may change
     over time.
 
+    .. include:: strict_list.rst
```

Comment:
Could you drop a comment here mentioning that the file is autogenerated?

Reply:
Good idea.